web/styles/zulip.css

```diff
@@ -837,6 +838,9 @@ strong {
 
             .typeahead-text-container {
                 align-self: center;
+                overflow: hidden;
+                text-overflow: ellipsis;
+                white-space: nowrap;
 
```

Comment:
Currently, only the stream description is shortened, and the descriptions for user groups and emails are left as it is in the typeahead. These CSS changes shorten those too and add `...` at the end. Is this OK or should I do this only for stream typeahead?

Reply:
I have tested that and it looks good:

<details><summary>Users</summary>
<p>

![Screenshot 2024-06-30 120420](https://github.com/zulip/zulip/assets/119798962/41e7be1f-7e51-42fe-9a7e-ef9105b72fbd)


</p>
</details> 

<details><summary>User groups</summary>
<p>

![Screenshot 2024-07-04 091223](https://github.com/zulip/zulip/assets/119798962/2754cf35-f749-4400-848d-686da02d601b)


</p>
</details>

Reply:
Yep, looks great!